Doc/library/functions.rst

```diff
@@ -825,6 +833,9 @@ are always available.  They are listed here in alphabetical order.
    Rather than being a function, :class:`list` is actually a mutable
    sequence type, as documented in :ref:`typesseq-list` and :ref:`typesseq`.
 
+   .. versionchanged:: 3.7
+      *x* is now a positional-only parameter.
```

Comment:
This change isn't needed, and would be wrong in any event. It's never been called "x", and the old keyword name, "sequence", was hidden in the C layer with no discoverability aside from reading the C source code, so you can't even make a sane versionchanged note ("iterable" was never a keyword parameter, and "sequence" has never been mentioned in the Python 3 docs, so mentioning it at all seems wildly out of place).